### PR TITLE
Feature/open camera function

### PR DESCRIPTION
**Pull request by BlueNealis**
What does this PR do?
-  Add an open Camera function
- Allow user to pick the image they take
-  Image populates on screen
Where should your reviewer start?
- Line 1 TakePicture.js
What (if any) features are you implementing?
- User can take a picture and see it on dashboard
What (if anything) did you refactor?
- N/A
Were there any issues that arose?
- Cannot select multiple images via imagePicker need to think of a manual gui solution
Is there anything that you need from your teammate?
- N/A
Any other comments, questions, or concerns?
- N/A 

#5

### CODE REVIEW

**Review by Neil-B-Linton — APPROVED** [review]
It looks good, great work!

File: components/TakePicture/TakePicture.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import * as ImagePicker from 'expo-image-picker'
+import * as Haptics from 'expo-haptics';
+import {styles} from '../PickImage/PickImageStyles'
+
+import { Button } from "react-native-paper"
+
+export default function TakePicture() {
+    const [localUri, setUri] = useState("")
+
+  let openCameraAsync = async () => {
```
Comment (sertmer) [inline components/TakePicture/TakePicture.js:13]: So cool to see yall taping into the camera, the haptics, and all of the other mobile specific options here. I Hope Native is treating you well!

File: components/PickImage/PickImageStyles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    backgroundColor: '#E4E4E4',
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  thumbnail: {
+    height: 200,
+    width: 200,
+    resizeMode: 'contain'
+  }
+});
```
Comment (sertmer) [inline components/PickImage/PickImageStyles.js:25]: Curious to hear the team's thoughts around the differences in styling organization between React and React Native! Do you have preferences for one or the other?

**Review by sertmer — COMMENTED** [review]
Looking good! Not a ton of feedback to give you, but it's great to see you diving in to `View`'s, `onPress`'s and style sheets here in React native!

**Pull request merged**